__tests__/utils/fonts.test.js

```diff
@@ -15,16 +15,11 @@ jest.mock('react-native', () => ({
 
 describe('Util font should correct', () => {
     it('normalize should run correctly', () => {
-        expect(normalize(10)).toEqual(8)
+        expect(normalize(10)).toEqual(9)
     })
 
    it('normalizeFontSize should run correctly', () => {
-        expect(normalizeFontSize(14)).toEqual(5.447470817120623)
+        expect(normalizeFontSize(14)).toEqual(12.600000000000001)
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The expected values for `normalize` and `normalizeFontSize` functions have changed. This suggests that the underlying logic of these functions has been altered. Ensure that these new values are correct and that they don't break any existing functionality that relies on these utility functions.
```diff
-        expect(normalize(10)).toEqual(8)
-        expect(normalizeFontSize(14)).toEqual(5.447470817120623)
+        expect(normalize(10)).toEqual(9)
+        expect(normalizeFontSize(14)).toEqual(12.600000000000001)
```
Also, consider using a precision or rounding function to avoid floating point precision issues like `12.600000000000001`.


<!-- This is an auto-generated comment by OSS CodeRabbit -->